ts_src/address.ts

```diff
@@ -53,7 +53,10 @@ function _toFutureSegwitAddress(output: Buffer, network: Network): string {
   return toBech32(data, version, network.bech32);
 }
 
-export function fromBase58Check(address: string): Base58CheckResult {
+export function fromBase58Check(
+  address: string,
+  bs58DecodeFunc: (address: string) => any = bs58check.decode,
```

Comment:
What's with the extra `any` in this type definition?

Reply:
This should be `BaseConverter['decode']` really.